2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1022,3 +1022,82 @@ Questa attività consiste nei seguenti compiti:
 - implementare e monitorare misure per valutare l'efficacia dei trattamenti dei rischi;
 
 - monitorare continuamente l'emergere di nuovi rischi e fonti durante l'intero ciclo di vita.
+
+
+== Processo di Controllo della Qualità
+
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.3.8_
+
+=== Scopo
+
+La International Software Testing Qualifications Board (ISTQB) definisce il controllo della qualità come "attività incentrate sul garantire che i requisiti di qualità saranno soddisfatti".
+
+Il processo di controllo della qualità ha quindi come obiettivo principale quello di garantire l'applicazione efficace del processo di gestione della qualità da parte del gruppo. 
+
+Il suo focus fondamentale è fornire la sicurezza e la certezza che i requisiti di qualità stabiliti saranno pienamente soddisfatti. 
+Questo viene ottenuto mediante un'analisi proattiva attenta dei processi e dei risultati che caratterizzano il ciclo di vita del progetto. 
+
+L'obiettivo è assicurare che il prodotto risponda alle aspettative in termini di qualità, e che ciò avvenga in piena conformità con le linee guida, le politiche e le procedure precedentemente definite. 
+In altre parole, la finalità ultima è garantire che la qualità desiderata sia intrinseca al prodotto finale e che ogni fase del processo segua gli standard e le disposizioni stabilite dal gruppo e dal progetto.
+
+=== Risultati
+
+L'implementazione riuscita del processo di controllo della qualità produce i seguenti risultati:
+
+- le procedure di controllo della qualità del progetto sono definite e implementate;
+- vengono definiti criteri e metodi per la valutazione di controllo della qualità;
+- vengono eseguite valutazioni dei prodotti, servizi e processi del progetto;
+- i risultati delle valutazioni vengono forniti agli stakeholders;
+- i problemi vengono trattati.
+
+=== Attività
+
+Le seguenti attività devono essere implementate in conformità con le politiche e le procedure del gruppo:
+
++ prepararsi per il controllo della qualità:
+
+   - definire una strategia di controllo della qualità. Essa consiste in: 
+
+      - ruoli, responsabilità e autorità definite:
+        - i ruoli e i compiti di ciascun membro sono definiti e non ambigui;
+        - ogni individuo fisico viene informato a scadenza bisettimanale dei propri compiti e delle proprie responsabilità per quel periodo di tempo;
+       
+      - criteri di valutazione definiti per processi, prodotti e servizi:
```

Comment:
- criteri di ... 
manca uno spazio prima del punto dell'elenco, viene visualizzata male in compilazione